jaxrs/runtime/src/main/java/org/wildfly/swarm/jaxrs/runtime/JAXRSConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package org.wildfly.swarm.jaxrs.runtime;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.xml.namespace.QName;
+
+import org.jboss.dmr.ModelNode;
+import org.jboss.staxmapper.XMLElementReader;
+import org.wildfly.swarm.jaxrs.JAXRSFraction;
+import org.wildfly.swarm.spi.runtime.AbstractParserFactory;
+import org.wildfly.swarm.spi.runtime.MarshallingServerConfiguration;
+
+/**
+ * @author Heiko Braun
+ * @since 23/03/16
+ */
+public class JAXRSConfiguration extends MarshallingServerConfiguration<JAXRSFraction> {
+
+    public static final String EXTENSION_MODULE = "org.jboss.as.jaxrs";
+
+    public JAXRSConfiguration() {
+        super(JAXRSFraction.class, EXTENSION_MODULE);
+    }
+
+    @Override
+    public Optional<Map<QName, XMLElementReader<List<ModelNode>>>> getSubsystemParsers() throws Exception {
+        return AbstractParserFactory.mapParserNamespaces(new JAXRSParserFactory());
+    }
+
```

Comment:
What's the benefit of changing 'jaxrs' from using @Configuration in api to having 'JAXRSConfiguration' in runtime?